pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel.go

```diff
@@ -26,19 +26,21 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+
 	"knative.dev/eventing/pkg/kncloudevents"
 
 	"go.uber.org/zap"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/reconciler"
 
+	"knative.dev/pkg/apis/duck"
```

Comment:
totes a nit, but these imports seem to be ordered a bit wonky, not your doing, but L30 should be below, well probably this too?